gatsby-plugin-material-ui/src/gatsby-node.js

```diff
@@ -29,7 +29,7 @@ exports.onPreBootstrap = ({ store, cache }, pluginOptions) => {
       module = module.split(`\\`).join(`\\\\`);
     }
   } else {
-    module = null;
+    module = `module.exports = null`;
```

Comment:
```suggestion
    module = 'module.exports = null';
```

Reply:
I followed the gatsby convention of using backticks for all strings(this file also uses it) for the sake of consistency. Are you sure you want me to change it?

Reply:
Only a suggestion, I'm not aware of this convention

Reply:
No worries, this rule is in eslintrc in this project btw, so I couldn't change it to single-quotes even if I wanted to https://github.com/hupe1980/gatsby-plugin-material-ui/blob/05588ae56ea3e9a7b2f5e4f1d2ad7234ba971d89/.eslintrc.js#L7